hbase_storage/src/main/java/com/splicemachine/si/impl/server/AbstractSICompactionScanner.java

```diff
@@ -84,7 +80,8 @@ public boolean next(List<Cell> list) throws IOException{
             compactionState.mutate(entry.cells, txns, list, purgeDeletedRows);
             if (!more) {
                 timer.cancel();
-                context.close();
+                if (context != null)
```

Comment:
Why are these guards needed?

Reply:
@dgomezferro Because I did not create a compaction context for table backup.

Reply:
Can you create one? I think it's cleaner to have it